picatrix/lib/manager.py

```diff
@@ -111,6 +171,28 @@ def get_magic_info(cls, as_pandas: bool = True) -> Union[
     df = pandas.DataFrame(entries)
     return df[cls.MAGICS_DF_COLUMNS].sort_values('name')
 
+  @classmethod
+  def register_helper(
+      cls, name: Text, helper: Any, typing_help: Dict[Text, Any]):
```

Comment:
Is there a way to better type the typing_help argument, something like Dict[Text, TypeHint] where TypeHint is  also defined somewhere?

Reply:
not sure, haven't found anything yet

Reply:
Ok, let's stick to Any then.